server/cli/src/cli.rs

```diff
@@ -435,6 +540,10 @@ async fn main() -> anyhow::Result<()> {
                 FormSchemaRowRepository::new(&con).upsert_one(form_schema_json)?;
             }
 
+            let is_custom = false;
+            let version = "1.0".to_string();
+            let code = None;
```

Comment:
This is for the upsert custom report (old report upsert method from oms reports repo). Maybe later we would want to have these as passable params?